src/cryptoadvance/specter/specter.py

```diff
@@ -119,6 +119,7 @@ def __init__(self, data_folder="./data", config={}):
             "proxy_url": "socks5h://localhost:9050",  # Tor proxy URL
             "only_tor": False,
             "tor_control_port": "",
+            "tor_status": "false",
```

Comment:
```suggestion
            "tor_status": False,
```
This will fix the first init issue. The problem is that you made it a string on init, which evaluates to true (if not empty) when checked as boolean. So instead of starting as false it started as string evaluating to False when first flipping it in  the `toggle_tor_status `

Reply:
Yes that was it. silly me. :)